src/coordinator/generated/assets/openapi/spec.yml

```diff
@@ -387,3 +417,60 @@ definitions:
     properties:
       error:
         type: "string"
+  DatabaseCreateRequest:
+    type: "object"
+    properties:
+      namespaceName:
+        type: "string"
+      type:
+        type: "string"
+        enum:
+        - "local"
+        - "cluster"
+      numShards:
+        type: "integer"
+        format: "int32"
+      replicationFactor:
+        type: "integer"
+        format: "int32"
+      retentionTime:
+        type: "string"
+      blockSize:
+        $ref: "#/definitions/BlockSize"
+      hosts:
+        type: "array"
+        items:
+          $ref: "#/definitions/Host"
+  BlockSize:
+    type: "object"
+    properties:
+      time:
+        type: "string"
+      expectedSeriesDatapointsPerHour:
+        type: "integer"
+        format: "int64"
+  Host:
+    type: "object"
+    properties:
+      id:
+        type: "string"
+      address:
+        type: "string"
+      port:
+        type: "integer"
+      isolationGroup:
```

Comment:
I wish there was a way to add a doc string to this since it might not be obvious what an `isolationGroup` means in this context.

Reply:
Agreed. Couldn't find a place for descriptions here that OpenAPI allows for. Perhaps best to augment this with other documentation as well like Github pages.